IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PowerControlSystem/Documentation.md

```diff
@@ -0,0 +1 @@
+Properties of power control system. The property set can be used by the predefined type ELECTRICAL of IfcDistributionSystem. The property set can be used to characterise the system that controls the railway energy network.
```

Comment:
```suggestion
Properties of power control system. The property set can be used by the predefined type ELECTRICAL of IfcDistributionSystem. The property set can be used to characterize the system that controls the railway energy network.
```

Reply:
Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/190/commits/be731fb1169a876200741df2c81db4c91e730337

Reply:
This one is not visible yet either.